Varia: Remove word-break style in tables

jacklenox opened the pull request:
Related to #3056.

The `word-break: break-all` style creates unwanted and unexpected breaks in text in tables. Let's remove it.

Fixes #2375, fixes #2120.

MaggieCabrera commented on alves/package.json:
```diff
@@ -1,6 +1,6 @@
 {
   "name": "alves",
-  "version": "1.4.14",
+  "version": "1.4.15",
```
I think we don't want to update the version numbers just yet, since there are more changes pending to be deployed to Varia children, we would update them once all are ready for deploy

MaggieCabrera commented:
Besides the issue with the version numbers, this LGTM

jacklenox commented:
> Besides the issue with the version numbers, this LGTM

Thanks @MaggieCabrera. Agh, yeah. The script does that automatically. I'm trying to think of a clever way of undoing those version numbers. I think I know how.

jacklenox commented:
(This was exactly the opposite of what I wanted to do. New commit incoming.)

Commit:
Varia: Remove word-break style in tables

Related to #3056.

The `word-break: break-all` style creates unwanted and unexpected breaks in text in tables. Let's remove it.

Fixes #2375, fixes #2120.
